src/components/Tab/Tab.js

```diff
@@ -28,14 +44,19 @@ const Tab = ({ onSelect, href, isSelected, children, description }) => {
 
 Tab.propTypes = {
   children: PropTypes.node.isRequired,
+  className: PropTypes.string,
   onSelect: PropTypes.func,
```

Comment:
should be required? I think so

Reply:
Ok, what to do... I need to extend ButtonHTMLAttributes/AnchorHTMLAttributes where onSelect is already declared. I think about removing it from here cause it should not be the main handler.

Reply:
Also It's not required 'cause `Tab` can be rendered as anchor element (there are some use cases in AA)

Reply:
Ok then remove onSelect and leave only onClick ;) that will resolve every problem here